docs/source/cli_help/list.rst

```diff
@@ -12,9 +12,9 @@ where <category> is one of: ``addable``, ``managed``, ``slidable``, ``slidable-a
 Lists all branches that fall into one of the specified categories:
     * ``addable``: all branches (local or remote) than can be added to the definition file,
     * ``managed``: all branches that appear in the definition file,
-    * ``slidable``: all managed branches that have an upstream and can be slid out with ``slide-out`` command
+    * ``slidable``: all managed branches that have an upstream and can be :ref:`slid out<slide-out>` with ``slide-out`` command
```

Comment:
Nit: :ref: just on `slide-out`? as with other places

Reply:
Ok, but we're losing the `preformatted` text this way, so there will be [blue link](blabla) instead of `in line code.`

Reply:
aaa okay leave as is then